environment-approvals.yml

```diff
@@ -11,11 +11,13 @@ prod:
   - repo: https://github.com/hmcts/ccd-api-gateway.git
   - repo: https://github.com/hmcts/ccd-case-activity-api.git
   - repo: https://github.com/hmcts/ccd-case-management-web.git
+  - repo: https://github.com/hmcts/ccd-case-ui-toolkit.git
```

Comment:
not sure if toolkit needs to be declared here since the pipeline we don't directly deploy it to Prod but release it to NPM instead. @timja?

Reply:
not needed

Reply:
ok will remove it